Sources/Ashton/AshtonHTMLWriter.swift

```diff
@@ -31,8 +31,9 @@ final class AshtonHTMLWriter {
                                                     paragraphTag.addAttributes(paragraphStyle)
 
                                                     if nsParagraphRange.length == nsrange.length {
+                                                        let nsString = attributedString.string as NSString
```

Comment:
- nit: we have the same variable defined as `string` in the else block - would extract that out of the if/else.
- do we still need `paragraphRange`, if we don't access it here anymore?